scripts/build/utils/MessageCodes.d.ts

```diff
@@ -0,0 +1,8 @@
+const enum MessageCodes {
```

Comment:
there is no real need to explicitly define the initializers, especially since we start at 0.
See https://www.typescriptlang.org/docs/handbook/enums.html#numeric-enums for more

Reply:
I think I did this because I was running into errors that needed me to explicitly declare the values. That was during some of my experimentation, so I'm not sure if the code that needed explicit enum value declarations is there anymore.